packages/wallet/src/useOnboard.tsx

```diff
@@ -44,7 +46,11 @@ const injectedWallets = new Set(['metamask', 'coinbase']);
 
 const injectedWallet = injectedModule();
 const ledger = ledgerModule();
-const walletConnect = walletConnectModule();
+export const walletConnect = walletConnectModule({
+    projectId: WALLETCONNECT_PROJECT_ID as string,
+    version: 2,
+    requiredChains: [1],
```

Comment:
@dandheedge, 
Here you could use the `Network` constants instead. That is used in the Chains definition below this line.